### PR TITLE
Responsive design

### PR DESCRIPTION
**Pull request by Yacubane**

### CODE REVIEW

File: src/components/Statistic.js [hunk 1]
```diff
@@ -11,7 +11,9 @@ class Statistic extends React.Component {
             return	<div className={styles.root}>
             <div className={styles.root2}>
```
Comment (piaskowyk) [inline src/components/Statistic.js]: What does mean "root2"? Is it possible to change this class name to more informative?

File: src/style/components/gauge-canvas.module.scss [hunk 2]
```diff
@@ -1,9 +1,13 @@
 .root {
-  position: "relative"
+  position: "relative";
+  width: 100%;
+  height: 100%;
 }
 
 .canvas {
   position: absolute;
```
Comment (piaskowyk) [inline src/style/components/gauge-canvas.module.scss:8]: Why we use here `position: absolute`?

File: src/style/components/gauge.module.scss [hunk 3]
```diff
@@ -1,11 +1,18 @@
...
+    min-width: 170px;
+}
+
+.root2 {
+    width: 100%;
+    padding-bottom: 70%;
 }
 
 .gauge {
     position: absolute;
```
Comment (piaskowyk) [inline src/style/components/gauge.module.scss:11]: like above

File: src/style/components/gauge.module.scss [hunk 4]
```diff
@@ -60,6 +67,7 @@
 }
 
 .name {
+    width: 100%;
     position: absolute;
```
Comment (piaskowyk) [inline src/style/components/gauge.module.scss:71]: Are you sure that `position: absolute` is here necessary?

File: src/style/components/statistic.module.scss [hunk 5]
```diff
@@ -9,31 +11,39 @@
...
     justify-content: center;
 }
 
-.leftIcon {
-    background: #DDDDDD;
+.leftIconBackgroundContainer {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    box-shadow: 0px 0px 0px 2px #AAAAAA inset;
```
Comment (piaskowyk) [inline src/style/components/statistic.module.scss]: Can we replace static hex color code with scss variable?

File: src/style/components/statistic.module.scss [hunk 6]
```diff
@@ -9,31 +11,39 @@
...
     border-radius: 50%;
     width: 50px;
     height: 50px;
 }
 
+.leftIcon {
+    width: 60%;
+    height: 60%;
+    color: #AAAAAA;
```
Comment (piaskowyk) [inline src/style/components/statistic.module.scss]: like above

File: src/style/main/components/SensorDetails.scss [hunk 7]
```diff
@@ -179,3 +190,32 @@
         }
     }
 }
+
+@media only screen and (max-width: 600px) {
```
Comment (piaskowyk) [inline src/style/main/components/SensorDetails.scss]: [Suggest] Do we trade with specific screen resolution? If yes, maybe we can specify standard screen resolution points in scss variables? 
I'm not sure what I said is a good idea. It's just suggestion.

File: src/views/main/components/SensorDetails.js [hunk 8]
```diff
@@ -181,8 +190,16 @@ class SensorDetails extends Component {
...
 				</div>
-				<div className="stationInside">
 				<ScrollBar className="stationList" options={{suppressScrollX : true}}>
+				<div className="close close-mobile">
+				<Button onClick={() => {
+					this.props.dispatch(sensorDetailAction(null));
+					this.setState({ visible: false });
+				}}>
+					
+					<FaRegTimesCircle className="closeIcon" size={22}></FaRegTimesCircle>
```
Comment (piaskowyk) [inline src/views/main/components/SensorDetails.js:200]: What do you think? Should we change static css class name to react's style import (`style.className`)?

File: src/components/Statistic.js (hunk 1 above)
Comment (Yacubane) [reply src/components/Statistic.js]: +1

File: src/style/components/gauge-canvas.module.scss (hunk 2 above)
Comment (Yacubane) [reply src/style/components/gauge-canvas.module.scss:8]: We have two canvases, one for background, one for actual progress - I use this style class for both of them to make sure that they are stacked together

Comment (piaskowyk) [reply src/style/components/gauge-canvas.module.scss:8]: ok, it's fine

File: src/style/main/components/SensorDetails.scss (hunk 7 above)
Comment (Yacubane) [reply src/style/main/components/SensorDetails.scss]: Good idea!

File: src/style/components/gauge.module.scss (hunk 4 above)
Comment (Yacubane) [reply src/style/components/gauge.module.scss:71]: I think that yes - name (like PM10) is being put onto canvas so these elements are overlapping. But maybe will it be better to use negative margins with position: static?

File: src/views/main/components/SensorDetails.js (hunk 8 above)
Comment (Yacubane) [reply src/views/main/components/SensorDetails.js:200]: We should :D

**Pull request merged**